packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -2386,4 +2386,58 @@ describe('ReactDOMFizzServer', () => {
         'Suspense boundary. Switched to client rendering.',
     ]);
   });
+
+  // @gate enableServerContext && experimental
+  it('supports ServerContext', async () => {
+    const {getOrCreateServerContext} = require('react/src/ReactServerContext');
```

Comment:
This is private implementation details which we shouldn't include in tests ideally. Can you restructure this test using only public APIs?